### ISSUE TITLE
Errors installing Laminas MVC Skeleton

### ISSUE THREAD

**Issue by charbugs** [laminas/laminas-mvc-skeleton#49]
laminas/laminas-mvc-skeleton (1.3.0 as well as 2.0.x-dev)
PHP 7.4.28
Composer 2.3.4
Debian GNU/Linux 11 (bullseye)

The install command from the [Laminas MVC tutorial](https://docs.laminas.dev/tutorials/getting-started/skeleton-application/) results in an error:

```
$ composer create-project -s dev laminas/laminas-mvc-skeleton path/to/install

...

[TypeError]                                                                                                                 
  Argument 1 passed to Symfony\Component\Console\Question\Question::__construct() must be of the type string, array given, c  
  alled in phar:///usr/local/bin/composer/src/Composer/IO/ConsoleIO.php on line 273

Exception trace:
  at phar:///usr/local/bin/composer/vendor/symfony/console/Question/Question.php:39
 Symfony\Component\Console\Question\Question->__construct() at phar:///usr/local/bin/composer/src/Composer/IO/ConsoleIO.php:273
 Composer\IO\ConsoleIO->ask() at /home/mathias/work/laminas-tutorial/vendor/laminas/laminas-skeleton-installer/src/OptionalPackagesInstaller.php:147
 Laminas\SkeletonInstaller\OptionalPackagesInstaller->requestMinimalInstall() at /home/mathias/work/laminas-tutorial/vendor/laminas/laminas-skeleton-installer/src/OptionalPackagesInstaller.php:75
 Laminas\SkeletonInstaller\OptionalPackagesInstaller->__invoke() at /home/mathias/work/laminas-tutorial/vendor/laminas/laminas-skeleton-installer/src/Plugin.php:76

...
```

[This article](https://ibexdataservices.com/news_errors-installing-laminas-mvc-skeleton/) describes the issue and provides a fix changing some lines of code. I tested the fix and it apparently solves the problem.

**Comment by Xerkus** [comment]
Issue is tracked at https://github.com/laminas/laminas-skeleton-installer/issues/31

**Comment by samsonasik** [comment]
The eeror seems different now:

```bash
composer create-project -sdev laminas/laminas-mvc-skeleton
Creating a "laminas/laminas-mvc-skeleton" project at "./laminas-mvc-skeleton"

In CreateProjectCommand.php line 435:
                                                                                                                                                                
  Could not find package laminas/laminas-mvc-skeleton with stability dev in a version installable using your PHP version, PHP extensions and Composer version.
```

**Comment by Xerkus** [comment]
Installing it from docker using `php:7.4` I see strange result for regular and for -s dev :
>  - Locking laminas/laminas-skeleton-installer (0.3.1)

Did we release skeleton after dropping the lock file? But for dev it should not be used anyway. quite strange tbh

**Comment by Xerkus** [comment]
doh! ^0.3 is not the same as ^1.3

**Comment by Ocramius** [comment]
Possibly solved by starting to run renovate here?

**Comment by Xerkus** [comment]
> Possibly solved by starting to run renovate here?

That is the idea. I will look at it tomorrow morning. 

I tested in docker on all versions and this issue should now be resolved with 1.4.0.  For php 8.1 `composer create-project -s dev laminas/laminas-mvc-skeleton` flag needs to be used until skeleton 2.0 is out.

**Comment by samsonasik** [comment]
I just re-tried it and it seems working ok now:

```
➜  www composer create-project -s dev laminas/laminas-mvc-skeleton
Creating a "laminas/laminas-mvc-skeleton" project at "./laminas-mvc-skeleton"
Info from https://repo.packagist.org: #StandWithUkraine
Installing laminas/laminas-mvc-skeleton (2.0.x-dev bef9fc7e6abd90df569380e73b6bea7cbf96fbfd)
  - Downloading laminas/laminas-mvc-skeleton (2.0.x-dev bef9fc7)
  - Installing laminas/laminas-mvc-skeleton (2.0.x-dev bef9fc7): Extracting archive
Created project in /Users/samsonasik/www/laminas-mvc-skeleton
Loading composer repositories with package information
Updating dependencies
Lock file operations: 22 installs, 0 updates, 0 removals
  - Locking brick/varexporter (0.3.5)
  - Locking container-interop/container-interop (1.2.0)
  - Locking laminas/laminas-component-installer (2.7.0)
  - Locking laminas/laminas-config (3.7.0)
  - Locking laminas/laminas-development-mode (3.6.0)
  - Locking laminas/laminas-escaper (2.10.0)
  - Locking laminas/laminas-eventmanager (3.5.0)
  - Locking laminas/laminas-http (2.15.1)
  - Locking laminas/laminas-json (3.3.0)
  - Locking laminas/laminas-loader (2.8.0)
  - Locking laminas/laminas-modulemanager (2.11.0)
  - Locking laminas/laminas-mvc (3.3.3)
  - Locking laminas/laminas-router (3.5.0)
  - Locking laminas/laminas-servicemanager (3.10.0)
  - Locking laminas/laminas-skeleton-installer (0.6.0)
  - Locking laminas/laminas-stdlib (3.7.1)
  - Locking laminas/laminas-uri (2.9.1)
  - Locking laminas/laminas-validator (2.17.0)
  - Locking laminas/laminas-view (2.20.0)
  - Locking nikic/php-parser (v4.13.2)
  - Locking psr/container (1.1.2)
  - Locking webimpress/safe-writer (2.2.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 22 installs, 0 updates, 0 removals
  - Downloading laminas/laminas-component-installer (2.7.0)
  - Downloading laminas/laminas-skeleton-installer (0.6.0)
  - Downloading nikic/php-parser (v4.13.2)
  - Downloading brick/varexporter (0.3.5)
  - Downloading psr/container (1.1.2)
  - Downloading laminas/laminas-stdlib (3.7.1)
  - Downloading laminas/laminas-config (3.7.0)
  - Downloading laminas/laminas-development-mode (3.6.0)
  - Downloading laminas/laminas-loader (2.8.0)
  - Downloading container-interop/container-interop (1.2.0)
  - Downloading laminas/laminas-servicemanager (3.10.0)
  - Downloading laminas/laminas-json (3.3.0)
  - Downloading laminas/laminas-eventmanager (3.5.0)
  - Downloading laminas/laminas-escaper (2.10.0)
  - Downloading laminas/laminas-view (2.20.0)
  - Downloading laminas/laminas-validator (2.17.0)
  - Downloading laminas/laminas-uri (2.9.1)
  - Downloading laminas/laminas-http (2.15.1)
  - Downloading laminas/laminas-router (3.5.0)
  - Downloading webimpress/safe-writer (2.2.0)
  - Downloading laminas/laminas-modulemanager (2.11.0)
  - Downloading laminas/laminas-mvc (3.3.3)
  - Installing laminas/laminas-component-installer (2.7.0): Extracting archive
  - Installing laminas/laminas-skeleton-installer (0.6.0): Extracting archive
  - Installing nikic/php-parser (v4.13.2): Extracting archive
  - Installing brick/varexporter (0.3.5): Extracting archive
  - Installing psr/container (1.1.2): Extracting archive
  - Installing laminas/laminas-stdlib (3.7.1): Extracting archive
  - Installing laminas/laminas-config (3.7.0): Extracting archive
  - Installing laminas/laminas-development-mode (3.6.0): Extracting archive
  - Installing laminas/laminas-loader (2.8.0): Extracting archive
  - Installing container-interop/container-interop (1.2.0): Extracting archive
  - Installing laminas/laminas-servicemanager (3.10.0): Extracting archive
  - Installing laminas/laminas-json (3.3.0): Extracting archive
  - Installing laminas/laminas-eventmanager (3.5.0): Extracting archive
  - Installing laminas/laminas-escaper (2.10.0): Extracting archive
  - Installing laminas/laminas-view (2.20.0): Extracting archive
  - Installing laminas/laminas-validator (2.17.0): Extracting archive
  - Installing laminas/laminas-uri (2.9.1): Extracting archive
  - Installing laminas/laminas-http (2.15.1): Extracting archive
  - Installing laminas/laminas-router (3.5.0): Extracting archive
  - Installing webimpress/safe-writer (2.2.0): Extracting archive
  - Installing laminas/laminas-modulemanager (2.11.0): Extracting archive
  - Installing laminas/laminas-mvc (3.3.3): Extracting archive
33 package suggestions were added by new dependencies, use `composer suggest` to see details.
Package container-interop/container-interop is abandoned, you should avoid using it. Use psr/container instead.
Generating autoload files
18 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

        Do you want a minimal install (no optional packages)? Y/n
n

    Would you like to install the developer toolbar? y/N
y
    Will install laminas/laminas-developer-tools (^2.1.1)
    When prompted to install as a module, select development.config.php.dist

    Would you like to install database support (installs laminas-db)? y/N
y
    Will install laminas/laminas-db (^2.12.0)
    When prompted to install as a module, select application.config.php or modules.config.php

    Would you like to install forms support? y/N
y
    Will install laminas/laminas-mvc-form (^2.0.0)
    When prompted to install as a module, select application.config.php or modules.config.php

    Would you like to install JSON de/serialization support? y/N
y
    Will install laminas/laminas-json (^3.2)

    Would you like to install logging support? y/N
y
    Will install laminas/laminas-log (^2.13.1)
    When prompted to install as a module, select application.config.php or modules.config.php

    Would you like to install command-line interface support? y/N
y
    Will install laminas/laminas-cli (^1.1.1)

    Would you like to install i18n support? y/N
y
    Will install laminas/laminas-mvc-i18n (^1.2.0)
    When prompted to install as a module, select application.config.php or modules.config.php

    Would you like to install the official MVC plugins, including PRG support, identity, and flash messages? y/N
y
    Will install laminas/laminas-mvc-plugins (^1.1.0)
    When prompted to install as a module, select application.config.php or modules.config.php

    Would you like to use the PSR-7 middleware dispatcher? y/N
y
    Will install laminas/laminas-mvc-middleware (^2.0.0)
    When prompted to install as a module, select application.config.php or modules.config.php

    Would you like to install sessions support? y/N
y
    Will install laminas/laminas-session (^2.10.0)
    When prompted to install as a module, select application.config.php or modules.config.php

    Would you like to install MVC testing tools for testing support? y/N
y
    Will install laminas/laminas-test (^4.0.0)

    Would you like to install the laminas-di for laminas-servicemanager? y/N
y
    Will install laminas/laminas-di (^3.2.2)
    When prompted to install as a module, select application.config.php or modules.config.php
Updating root package
    Running an update to install optional packages
Loading composer repositories with package information
Updating dependencies
Lock file operations: 80 installs, 0 updates, 0 removals
  - Locking doctrine/annotations (1.13.2)
  - Locking doctrine/instantiator (1.4.1)
  - Locking doctrine/lexer (1.2.3)
  - Locking fig/http-message-util (1.1.5)
  - Locking laminas/laminas-authentication (2.10.1)
  - Locking laminas/laminas-cli (1.4.0)
  - Locking laminas/laminas-db (2.15.0)
  - Locking laminas/laminas-developer-tools (2.3.0)
  - Locking laminas/laminas-di (3.4.0)
  - Locking laminas/laminas-diactoros (2.9.2)
  - Locking laminas/laminas-filter (2.14.0)
  - Locking laminas/laminas-form (3.1.1)
  - Locking laminas/laminas-hydrator (4.3.1)
  - Locking laminas/laminas-i18n (2.15.0)
  - Locking laminas/laminas-inputfilter (2.13.0)
  - Locking laminas/laminas-log (2.15.0)
  - Locking laminas/laminas-mvc-form (2.0.0)
  - Locking laminas/laminas-mvc-i18n (1.3.1)
  - Locking laminas/laminas-mvc-middleware (2.2.0)
  - Locking laminas/laminas-mvc-plugin-fileprg (1.3.1)
  - Locking laminas/laminas-mvc-plugin-flashmessenger (1.7.0)
  - Locking laminas/laminas-mvc-plugin-identity (1.3.0)
  - Locking laminas/laminas-mvc-plugin-prg (1.5.0)
  - Locking laminas/laminas-mvc-plugins (1.2.0)
  - Locking laminas/laminas-psr7bridge (1.6.0)
  - Locking laminas/laminas-session (2.12.1)
  - Locking laminas/laminas-stratigility (3.5.0)
  - Locking laminas/laminas-test (4.0.0)
  - Locking myclabs/deep-copy (1.11.0)
  - Locking phar-io/manifest (2.0.3)
  - Locking phar-io/version (3.2.1)
  - Locking phpdocumentor/reflection-common (2.2.0)
  - Locking phpdocumentor/reflection-docblock (5.3.0)
  - Locking phpdocumentor/type-resolver (1.6.1)
  - Locking phpspec/prophecy (v1.15.0)
  - Locking phpunit/php-code-coverage (9.2.15)
  - Locking phpunit/php-file-iterator (3.0.6)
  - Locking phpunit/php-invoker (3.1.1)
  - Locking phpunit/php-text-template (2.0.4)
  - Locking phpunit/php-timer (5.0.3)
  - Locking phpunit/phpunit (9.5.20)
  - Locking psr/cache (3.0.0)
  - Locking psr/event-dispatcher (1.0.0)
  - Locking psr/http-factory (1.0.1)
  - Locking psr/http-message (1.0.1)
  - Locking psr/http-server-handler (1.0.1)
  - Locking psr/http-server-middleware (1.0.1)
  - Locking psr/log (1.1.4)
  - Locking sebastian/cli-parser (1.0.1)
  - Locking sebastian/code-unit (1.0.8)
  - Locking sebastian/code-unit-reverse-lookup (2.0.3)
  - Locking sebastian/comparator (4.0.6)
  - Locking sebastian/complexity (2.0.2)
  - Locking sebastian/diff (4.0.4)
  - Locking sebastian/environment (5.1.4)
  - Locking sebastian/exporter (4.0.4)
  - Locking sebastian/global-state (5.0.5)
  - Locking sebastian/lines-of-code (1.0.3)
  - Locking sebastian/object-enumerator (4.0.4)
  - Locking sebastian/object-reflector (2.0.4)
  - Locking sebastian/recursion-context (4.0.4)
  - Locking sebastian/resource-operations (3.0.3)
  - Locking sebastian/type (3.0.0)
  - Locking sebastian/version (3.0.2)
  - Locking symfony/console (v6.0.7)
  - Locking symfony/css-selector (v6.0.3)
  - Locking symfony/deprecation-contracts (v3.0.1)
  - Locking symfony/dom-crawler (v6.0.6)
  - Locking symfony/event-dispatcher (v6.0.3)
  - Locking symfony/event-dispatcher-contracts (v3.0.1)
  - Locking symfony/polyfill-ctype (v1.25.0)
  - Locking symfony/polyfill-intl-grapheme (v1.25.0)
  - Locking symfony/polyfill-intl-normalizer (v1.25.0)
  - Locking symfony/polyfill-mbstring (v1.25.0)
  - Locking symfony/polyfill-php80 (v1.25.0)
  - Locking symfony/service-contracts (v2.5.1)
  - Locking symfony/string (v6.0.3)
  - Locking symfony/var-dumper (v6.0.6)
  - Locking theseer/tokenizer (1.2.1)
  - Locking webmozart/assert (1.10.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 80 installs, 0 updates, 0 removals
  - Downloading doctrine/lexer (1.2.3)
  - Downloading laminas/laminas-authentication (2.10.1)
  - Downloading webmozart/assert (1.10.0)
  - Downloading psr/event-dispatcher (1.0.0)
  - Downloading symfony/event-dispatcher (v6.0.3)
  - Downloading symfony/polyfill-intl-normalizer (v1.25.0)
  - Downloading symfony/polyfill-intl-grapheme (v1.25.0)
  - Downloading symfony/string (v6.0.3)
  - Downloading symfony/service-contracts (v2.5.1)
  - Downloading laminas/laminas-cli (1.4.0)
  - Downloading laminas/laminas-db (2.15.0)
  - Downloading symfony/var-dumper (v6.0.6)
  - Downloading laminas/laminas-developer-tools (2.3.0)
  - Downloading psr/log (1.1.4)
  - Downloading laminas/laminas-di (3.4.0)
  - Downloading laminas/laminas-hydrator (4.3.1)
  - Downloading laminas/laminas-filter (2.14.0)
  - Downloading laminas/laminas-inputfilter (2.13.0)
  - Downloading laminas/laminas-log (2.15.0)
  - Downloading laminas/laminas-i18n (2.15.0)
  - Downloading laminas/laminas-form (3.1.1)
  - Downloading psr/cache (3.0.0)
  - Downloading doctrine/annotations (1.13.2)
  - Downloading laminas/laminas-mvc-i18n (1.3.1)
  - Downloading psr/http-message (1.0.1)
  - Downloading psr/http-server-handler (1.0.1)
  - Downloading psr/http-server-middleware (1.0.1)
  - Downloading fig/http-message-util (1.1.5)
  - Downloading laminas/laminas-stratigility (3.5.0)
  - Downloading psr/http-factory (1.0.1)
  - Downloading laminas/laminas-diactoros (2.9.2)
  - Downloading laminas/laminas-psr7bridge (1.6.0)
  - Downloading laminas/laminas-mvc-middleware (2.2.0)
  - Downloading laminas/laminas-session (2.12.1)
  - Downloading laminas/laminas-mvc-plugin-prg (1.5.0)
  - Downloading laminas/laminas-mvc-plugin-identity (1.3.0)
  - Downloading laminas/laminas-mvc-plugin-flashmessenger (1.7.0)
  - Downloading laminas/laminas-mvc-plugin-fileprg (1.3.1)
  - Downloading symfony/dom-crawler (v6.0.6)
  - Downloading symfony/css-selector (v6.0.3)
  - Downloading sebastian/version (3.0.2)
  - Downloading sebastian/type (3.0.0)
  - Downloading sebastian/resource-operations (3.0.3)
  - Downloading sebastian/recursion-context (4.0.4)
  - Downloading sebastian/object-reflector (2.0.4)
  - Downloading sebastian/object-enumerator (4.0.4)
  - Downloading sebastian/global-state (5.0.5)
  - Downloading sebastian/exporter (4.0.4)
  - Downloading sebastian/diff (4.0.4)
  - Downloading sebastian/comparator (4.0.6)
  - Downloading sebastian/code-unit (1.0.8)
  - Downloading sebastian/cli-parser (1.0.1)
  - Downloading phpunit/php-timer (5.0.3)
  - Downloading phpunit/php-text-template (2.0.4)
  - Downloading phpunit/php-invoker (3.1.1)
  - Downloading phpunit/php-file-iterator (3.0.6)
  - Downloading theseer/tokenizer (1.2.1)
  - Downloading sebastian/lines-of-code (1.0.3)
  - Downloading sebastian/complexity (2.0.2)
  - Downloading sebastian/code-unit-reverse-lookup (2.0.3)
  - Downloading phpunit/php-code-coverage (9.2.15)
  - Downloading phpdocumentor/reflection-common (2.2.0)
  - Downloading phpdocumentor/reflection-docblock (5.3.0)
  - Downloading doctrine/instantiator (1.4.1)
  - Downloading phpspec/prophecy (v1.15.0)
  - Downloading phar-io/version (3.2.1)
  - Downloading phar-io/manifest (2.0.3)
  - Downloading myclabs/deep-copy (1.11.0)
  - Downloading laminas/laminas-test (4.0.0)
  - Installing doctrine/lexer (1.2.3): Extracting archive
  - Installing laminas/laminas-authentication (2.10.1): Extracting archive
  - Installing symfony/polyfill-ctype (v1.25.0): Extracting archive
  - Installing webmozart/assert (1.10.0): Extracting archive
  - Installing symfony/polyfill-php80 (v1.25.0): Extracting archive
  - Installing psr/event-dispatcher (1.0.0): Extracting archive
  - Installing symfony/event-dispatcher-contracts (v3.0.1): Extracting archive
  - Installing symfony/event-dispatcher (v6.0.3): Extracting archive
  - Installing symfony/polyfill-mbstring (v1.25.0): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.25.0): Extracting archive
  - Installing symfony/polyfill-intl-grapheme (v1.25.0): Extracting archive
  - Installing symfony/string (v6.0.3): Extracting archive
  - Installing symfony/deprecation-contracts (v3.0.1): Extracting archive
  - Installing symfony/service-contracts (v2.5.1): Extracting archive
  - Installing symfony/console (v6.0.7): Extracting archive
  - Installing laminas/laminas-cli (1.4.0): Extracting archive
  - Installing laminas/laminas-db (2.15.0): Extracting archive
  - Installing symfony/var-dumper (v6.0.6): Extracting archive
  - Installing laminas/laminas-developer-tools (2.3.0): Extracting archive
  - Installing psr/log (1.1.4): Extracting archive
  - Installing laminas/laminas-di (3.4.0): Extracting archive
  - Installing laminas/laminas-hydrator (4.3.1): Extracting archive
  - Installing laminas/laminas-filter (2.14.0): Extracting archive
  - Installing laminas/laminas-inputfilter (2.13.0): Extracting archive
  - Installing laminas/laminas-log (2.15.0): Extracting archive
  - Installing laminas/laminas-i18n (2.15.0): Extracting archive
  - Installing laminas/laminas-form (3.1.1): Extracting archive
  - Installing psr/cache (3.0.0): Extracting archive
  - Installing doctrine/annotations (1.13.2): Extracting archive
  - Installing laminas/laminas-mvc-form (2.0.0)
  - Installing laminas/laminas-mvc-i18n (1.3.1): Extracting archive
  - Installing psr/http-message (1.0.1): Extracting archive
  - Installing psr/http-server-handler (1.0.1): Extracting archive
  - Installing psr/http-server-middleware (1.0.1): Extracting archive
  - Installing fig/http-message-util (1.1.5): Extracting archive
  - Installing laminas/laminas-stratigility (3.5.0): Extracting archive
  - Installing psr/http-factory (1.0.1): Extracting archive
  - Installing laminas/laminas-diactoros (2.9.2): Extracting archive
  - Installing laminas/laminas-psr7bridge (1.6.0): Extracting archive
  - Installing laminas/laminas-mvc-middleware (2.2.0): Extracting archive
  - Installing laminas/laminas-session (2.12.1): Extracting archive
  - Installing laminas/laminas-mvc-plugin-prg (1.5.0): Extracting archive
  - Installing laminas/laminas-mvc-plugin-identity (1.3.0): Extracting archive
  - Installing laminas/laminas-mvc-plugin-flashmessenger (1.7.0): Extracting archive
  - Installing laminas/laminas-mvc-plugin-fileprg (1.3.1): Extracting archive
  - Installing laminas/laminas-mvc-plugins (1.2.0)
  - Installing symfony/dom-crawler (v6.0.6): Extracting archive
  - Installing symfony/css-selector (v6.0.3): Extracting archive
  - Installing sebastian/version (3.0.2): Extracting archive
  - Installing sebastian/type (3.0.0): Extracting archive
  - Installing sebastian/resource-operations (3.0.3): Extracting archive
  - Installing sebastian/recursion-context (4.0.4): Extracting archive
  - Installing sebastian/object-reflector (2.0.4): Extracting archive
  - Installing sebastian/object-enumerator (4.0.4): Extracting archive
  - Installing sebastian/global-state (5.0.5): Extracting archive
  - Installing sebastian/exporter (4.0.4): Extracting archive
  - Installing sebastian/environment (5.1.4): Extracting archive
  - Installing sebastian/diff (4.0.4): Extracting archive
  - Installing sebastian/comparator (4.0.6): Extracting archive
  - Installing sebastian/code-unit (1.0.8): Extracting archive
  - Installing sebastian/cli-parser (1.0.1): Extracting archive
  - Installing phpunit/php-timer (5.0.3): Extracting archive
  - Installing phpunit/php-text-template (2.0.4): Extracting archive
  - Installing phpunit/php-invoker (3.1.1): Extracting archive
  - Installing phpunit/php-file-iterator (3.0.6): Extracting archive
  - Installing theseer/tokenizer (1.2.1): Extracting archive
  - Installing sebastian/lines-of-code (1.0.3): Extracting archive
  - Installing sebastian/complexity (2.0.2): Extracting archive
  - Installing sebastian/code-unit-reverse-lookup (2.0.3): Extracting archive
  - Installing phpunit/php-code-coverage (9.2.15): Extracting archive
  - Installing phpdocumentor/reflection-common (2.2.0): Extracting archive
  - Installing phpdocumentor/type-resolver (1.6.1): Extracting archive
  - Installing phpdocumentor/reflection-docblock (5.3.0): Extracting archive
  - Installing doctrine/instantiator (1.4.1): Extracting archive
  - Installing phpspec/prophecy (v1.15.0): Extracting archive
  - Installing phar-io/version (3.2.1): Extracting archive
  - Installing phar-io/manifest (2.0.3): Extracting archive
  - Installing myclabs/deep-copy (1.11.0): Extracting archive
  - Installing phpunit/phpunit (9.5.20): Extracting archive
  - Installing laminas/laminas-test (4.0.0): Extracting archive

  Please select which config file you wish to inject 'Laminas\Db' into:
  [0] Do not inject
  [1] config/modules.config.php
  [2] config/development.config.php.dist
  Make your selection (default is 1):1

  Remember this option for other packages of the same type? (Y/n)y
    Installing Laminas\Db from package laminas/laminas-db

  Please select which config file you wish to inject 'Laminas\DeveloperTools' into:
  [0] Do not inject
  [1] config/modules.config.php
  [2] config/development.config.php.dist
  Make your selection (default is 1):2

  Remember this option for other packages of the same type? (Y/n)Y
    Installing Laminas\DeveloperTools from package laminas/laminas-developer-tools
    Installing Laminas\Di from package laminas/laminas-di
    Installing Laminas\Hydrator from package laminas/laminas-hydrator
    Installing Laminas\Filter from package laminas/laminas-filter
    Installing Laminas\InputFilter from package laminas/laminas-inputfilter
    Installing Laminas\Log from package laminas/laminas-log
    Installing Laminas\I18n from package laminas/laminas-i18n
    Installing Laminas\Form from package laminas/laminas-form
    Installing Laminas\Mvc\I18n from package laminas/laminas-mvc-i18n
    Installing Laminas\Diactoros from package laminas/laminas-diactoros
    Installing Laminas\Mvc\Middleware from package laminas/laminas-mvc-middleware
    Installing Laminas\Session from package laminas/laminas-session
    Installing Laminas\Mvc\Plugin\Prg from package laminas/laminas-mvc-plugin-prg
    Installing Laminas\Mvc\Plugin\Identity from package laminas/laminas-mvc-plugin-identity
    Installing Laminas\Mvc\Plugin\FlashMessenger from package laminas/laminas-mvc-plugin-flashmessenger
    Installing Laminas\Mvc\Plugin\FilePrg from package laminas/laminas-mvc-plugin-fileprg
28 package suggestions were added by new dependencies, use `composer suggest` to see details.
Package container-interop/container-interop is abandoned, you should avoid using it. Use psr/container instead.
Generating autoload files
82 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
    Updating composer.json
> laminas-development-mode enable
You are now in development mode.
> php bin/update-gitignore.php
> php -r 'if (file_exists("bin/remove-package-artifacts.php")) include "bin/remove-package-artifacts.php";'
> php -r 'if (file_exists("CHANGELOG.md")) unlink("CHANGELOG.md");'
➜  www cd laminas-mvc-skeleton 
➜  laminas-mvc-skeleton composer serve
> php -S 0.0.0.0:8080 -t public
[Sun Apr 17 21:07:28 2022] PHP 8.1.4 Development Server (http://0.0.0.0:8080) started
```

I am closing it.
